qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -294,14 +292,16 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
     }
 
     def triggerProcessViolation(SplunkUtil.SplunkDeployment splunkDeployment) {
-        orchestrator.execInContainer(splunkDeployment.deployment, "curl http://127.0.0.1:10248/")
+        orchestrator.execInContainer(splunkDeployment.deployment, "curl http://127.0.0.1:10248/ --max-time 2")
         assert waitForAlertWithPolicyId("86804b96-e87e-4eae-b56e-1718a8a55763")
     }
 
     def triggerNetworkFlowViolation(SplunkUtil.SplunkDeployment splunkDeployment, String centralHost) {
-        NetworkBaselineService.lockNetworkBaseline(splunkDeployment.getDeployment().deploymentUid)
+        def uid = splunkDeployment.getDeployment().getDeploymentUid()
+        NetworkBaselineService.getNetworkBaseline(uid)
+        NetworkBaselineService.lockNetworkBaseline(uid)
         orchestrator.execInContainer(splunkDeployment.deployment,
-                "for i in `seq 25`; do curl http://${centralHost}:443; sleep 1; done")
+                "for i in `seq 100`; do  wget -S http://${centralHost}:443; sleep 1; done")
```

Comment:
Why `wget` instead of `curl`?

Reply:
I wish I could tell you why that one uses wget, but it was like that before I showed up.